packages/shared/src/components/tooltips/LazyTooltip.module.css

```diff
@@ -0,0 +1,159 @@
+@keyframes fade-in-top {
+  0% {
+    opacity: 0;
+    transform: translateY(0.1875rem);
```

Comment:
Also would suggest to look into translate3d it does the same thing, but uses GPU over CPU, meaning the animation are a bit more cost effective.
You could also use the `will-change` property.

The following stack overflow has some good tips on performance for animations:
https://stackoverflow.com/questions/13176746/css-keyframe-animation-cpu-usage-is-high-should-it-be-this-way

Reply:
I have now applied the `will-change` property, thank you so much for the tip! I learned today!